src/home/components/home-banner/HomeBanner.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+import { getConfig } from '@edx/frontend-platform';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Button, Form, IconButton, ModalDialog, useToggle,
+} from '@openedx/paragon';
+import { Search as SearchIcon } from '@openedx/paragon/icons';
+import { useNavigate } from 'react-router';
+
+import messages from './messages';
+
+const HomeBanner = () => {
+  const intl = useIntl();
+  const navigate = useNavigate();
+  const [searchValue, setSearchValue] = useState('');
+  const [isOpen, open, close] = useToggle(false);
+
+  const {
+    HOMEPAGE_OVERLAY_HTML,
+    SITE_NAME,
+    ENABLE_HOMEPAGE_COURSE_DISCOVERY,
+    HOMEPAGE_PROMO_VIDEO_YOUTUBE_ID,
+  } = getConfig();
+
+  const handleSearch = () => navigate(`/courses?search_query=${searchValue}`);
+
+  return (
+    <section className="heading" data-testid="home-banner">
+      <div className="outer-wrapper">
+        {HOMEPAGE_OVERLAY_HTML ? (
+          <h1
+            className="display-1 text-white text-center"
+            dangerouslySetInnerHTML={{ __html: HOMEPAGE_OVERLAY_HTML }}
+          />
```

Comment:
[nit] The `HOMEPAGE_OVERLAY_HTML` might contain any HTML for render so we must render that HTML as is. Something like:
```suggestion
          <div dangerouslySetInnerHTML={{ __html: HOMEPAGE_OVERLAY_HTML }} />
```

Reply:
fixed